docs/application/get-started/mobile.md

```diff
@@ -1,40 +1,49 @@
-# Get Started for Mobile
 
-The Get Started Tutorial teaches you how to create basic Tizen applications for mobile devices.
+# Get Started with Tizen Mobile Applications
+Samsung expands the Tizen ecosystem with advanced Tizen-based smart phone application development.  
 
-The Tizen supports three primary application types for mobile:
+## Mobile Native Applications
+You can create rich applications in C with device-specific features, advanced system settings, and additional functionalities.
 
--   [.NET application](#net-application) is a new way to develop applications for the Tizen operating system using C# language.
+[Quick start](../native/get-started/mobile/first-app.md) for beginners
 
--   [Native application](#native-application) is developed using C language. Developer can use the dozens of native API modules that cover a large range of capabilities such as camera, GPS, and accelerometer in addition to more advanced system settings and functionality.
+Explore more about the Tizen mobile Native application development:
+- [Tutorials](../native/tutorials/overview.md)
+- [Guides](../native/guides/index.md)
+- [API reference](../native/api/overview.md)
+- [Samples](https://developer.tizen.org/development/sample/native)
```

Comment:
Global Comment: 
There is some mismatch with some links. I will update the relative links as soon as possible.
For now, updated Tizen website links.